dao/src/main/java/greencity/repository/DeactivateChosenEntityRepository.java

```diff
@@ -188,6 +188,71 @@ public void deactivateTariffsByRegionAndCitiesAndStations(Long regionId, List<Lo
             .forEach(tariffsInfo -> tariffsInfo.setLocationStatus(LocationStatus.DEACTIVATED));
     }
 
+    /**
+     * Method that deactivate tariffs for region id, list of cities id , list of
+     * station id and courier id.
```

Comment:
Javadoc does not describe what the method does

Reply:
changed